src/common/QskRgbLiterals.h

```diff
@@ -0,0 +1,142 @@
+/******************************************************************************
+ * Copyright (C) 2023 Edelhirsch Software GmbH
+ * This file may be used under the terms of the 3-clause BSD License
+ *****************************************************************************/
+
+#ifndef QSK_RGB_LITERALS_H
+#define QSK_RGB_LITERALS_H
+
+namespace QskRgb
+{
+    namespace literals
+    {
+        /// @brief converts HTML #RRGGBB[AA] color hex string to Qt's RGB 0xAARRGGBB integer
```

Comment:
Please write the comments as plain text - the doxygen parser will never see this file and using its syntax makes the documentation less readable.

The doxygen generated documentation will be generated from files in the doc folder only - and yes this approach is against the basic idea of having implementation and documentation in the same place.